lib/matplotlib/projections/polar.py

```diff
@@ -686,6 +690,46 @@ def drag_pan(self, button, key, x, y):
             scale = r / startr
             self.set_rmax(p.rmax / scale)
 
+    def set_r_offset(self, input_offset):
+        """
+        Creates an offset from the origin of a polar plot
+        
+        Parameters
+        ----------
+        input_offset : float
+              A number between 0 and 1 where 0 is the center of the plot and 1
+              is the edge of the plot
+        
+        Returns
+        -------
+        No return value.              
+        """
+        if input_offset < 0 or input_offset >= 1:
+            raise NotImplementedError("Offset must be greater than 0 and less than 1.")
+        elif input_offset == 0:
+            return
+        rmax = self.get_rmax()
+        rmin = self.get_rmin()
+        rrange = rmax-rmin
+        offset = rmin-(input_offset*rrange)/(1.0-input_offset)
+        self.set_rmin(offset)
+        self._r_offset = input_offset
+        self.add_artist(Wedge((0.5,0.5), input_offset*.5, 0,360, transform=self.transAxes, color=self.patch.get_facecolor(),zorder=99,ec='k'))
```

Comment:
Does this just mask out the center of the graph?  Calling this function more than once will result in an accumulation of artists.